MetafileImporter/qSlicerMetafileReader.cxx

```diff
@@ -86,7 +87,7 @@ qSlicerIO::IOFileType qSlicerMetafileReader::fileType() const
 //-----------------------------------------------------------------------------
 QStringList qSlicerMetafileReader::extensions() const
 {
-  return QStringList() << "Sequence Metafile (*.seq.mha *.seq.mhd *.mha *.mhd)";
+  return QStringList() << "Sequence Metafile (*.seq.mha *.seq.mhd *.mha *.mhd *.seq.nrrd *.nrrd)";
```

Comment:
seq.nrrd extension is already designated as volume sequence. If we add this file extension here then by default seq.nrrd files will be either offered to be loaded as volume sequence or as sequence metafile.

One solution could be to further complicate the extension, such as:
.plus.seq.nrrd
.igsio.seq.nrrd
.igs.seq.nrrd
.trk.seq.nrrd
...

Another option could be to specify a new composite file extension:
.igs.nrrd, .igs.mha
.igsio.nrrd, .igsio.mha
.trk.nrrd, .trk.mha
.plus.nrrd, .plus.mha